pkg/network/ebpf/c/protocols/kafka/defs.h

```diff
@@ -50,4 +50,7 @@
 // The size of each kafka telemetry topic name bucket
 #define KAFKA_TELEMETRY_TOPIC_NAME_BUCKET_SIZE 10
 
+// The maximum version we'll support in telemetry.
+#define KAFKA_TELEMETRY_API_VERSIONS_NUM_OF_BUCKETS 30
```

Comment:
not sure why `30`, and how is that the max we can support
as we support fetch (v17) and produce (v12), why do we need 30 entries?